tilelang/profiler/bench.py

```
@@ -98,3 +100,171 @@ def do_bench(
+def do_bench(
+    fn: Callable,
+    warmup: float = 25,
+    rep: float = 100,
+    _n_warmup: int = 0,
+    _n_repeat: int = 0,
+    grad_to_none: Optional[List[torch.Tensor]] = None,
+    quantiles: Optional[List[float]] = None,
+    fast_flush: bool = True,
+    backend: Literal["event", "cupti"] = "event",
+    return_mode: Literal["min", "max", "mean", "median"] = "mean",
+    trace_path: str = None,
```

Ruff (0.13.3)

PEP 484 prohibits implicit `Optional`

Convert to `Optional[T]`

(RUF013)